helm-gtags.el

```diff
@@ -501,6 +501,48 @@ Always update if value of this variable is nil."
     (setq helm-gtags--last-input input)
     (reverse (cons input options))))
 
+(defun helm-gtgs--exec-global-command-local (type input)
+  (with-temp-buffer
+    (let ((args (reverse (cons input (helm-gtags--construct-options type nil)))))
+      (progn
+        (push "-a" args)
+        (apply 'process-file "global" nil '(t nil) nil args)
+        (buffer-substring-no-properties (point-min) (point-max))))))
+
+(defun helm-gtags--which-function-only-func-name ()
+  (if (string-match "lisp-mode$" (symbol-name major-mode))
+      (which-function)
+    (save-excursion
+      (progn
+        (beginning-of-defun)
+        (when (re-search-forward "(" nil t)
+          (forward-char -1)
+          (let ((end (point)))
+            (skip-chars-backward "_A-Za-z0-9")
+            (buffer-substring-no-properties (point) end)))))))
+
+(defun helm-gtags--tag-included-in-bound (bound tag-line)
+  (ignore-errors
+    (let* ((tag-elms (split-string tag-line ":"))
+           (tag-file-name (car tag-elms))
+           (line-num (string-to-int (car (cdr tag-elms)))))
```

Comment:
`string-to-int` is obsoleted. You should use `string-to-number`.
And you should use `cadr` or `cl-second` instead of `cdr` + `car`.